applications/dolphin/passport/passport.c

```diff
@@ -103,8 +103,7 @@ int32_t passport_app(void* p) {
     gui_add_view_port(gui, view_port, GuiLayerFullscreen);
     view_port_update(view_port);
 
-    osStatus_t status = osSemaphoreAcquire(semaphore, osWaitForever);
-    furi_assert(status == osOK);
+    furi_check(osSemaphoreAcquire(semaphore, osWaitForever) == osOK);
```

Comment:
Do we really need a check here? I suggest assert and mark osStatus_t status as probably unused. (I mean __attribute__((unused)))

Reply:
It was resolved internally.

Reply:
Whole thing will be gone with CMSIS, till then all primitives provided by CMSIS treated as unsafe and guarded with furi_check.